ext/curl/tests/curl_string_file_upload.phpt

```diff
@@ -0,0 +1,54 @@
+--TEST--
+CURL file uploading from string
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+function testcurl($ch, $postname, $data, $mime = '')
```

Comment:
This should `$mime = null` and pass it unconditionally rather than checking `empty`